tasks/buildah.yml

```diff
@@ -0,0 +1,85 @@
+---
```

Comment:
Why it is so scrambled in comparison with the original task? It is hard to compare it with the original task so that somebody in the future could have problems understanding what was changed here.
Could you also please add some small note, mentioning the source of the original task (https://github.com/tektoncd/catalog/blob/main/task/buildah/0.2/buildah.yaml) and what was changed there?

Reply:
The buildah task versions that are available in tekton hub don't work on our clusters because it requires a privileged access. That is why I took the buildah task directly from `ClusterTask` from a cluster and just replaced `ClusterTask` with `Task`

Reply:
Still, just a small part of the logic is changed compared to the tektonhub task,  but a large part of the definition is changed (eg. in array "params"
Original:
```
  params:
  - name: IMAGE
    description: Reference of the image buildah will produce.
  ```
  
Our definition:
  ```
      - description: Reference of the image buildah will produce.
      name: IMAGE
      type: string
```

Reply:
After discussion with @Allda, he will create a task to push the changes in the Buildah task definition in tekton hub, and then we'll replace the task definition